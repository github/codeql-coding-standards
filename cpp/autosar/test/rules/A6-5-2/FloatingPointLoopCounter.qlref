rules/A6-5-2/FloatingPointLoopCounter.ql