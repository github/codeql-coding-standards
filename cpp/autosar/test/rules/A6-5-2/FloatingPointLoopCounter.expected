| test.cpp:2:14:2:14 | f | Loop iteration variable f has a floating-point type. |
